src/commands/android/subcommands/common.ts

```diff
@@ -0,0 +1,74 @@
+import colors from 'ansi-colors';
+
+import Logger from '../../../logger';
+import {AVAILABLE_SUBCOMMANDS} from '../constants';
+import {Options} from '../interfaces';
+import {getSubcommandOptionsHelp} from '../utils/common';
+
+export function showHelp(subcommand: string) {
+  const subcmd = AVAILABLE_SUBCOMMANDS[subcommand];
+
+  Logger.log(`Usage: ${colors.cyan(`npx @nightwatch/mobile-helper android ${subcommand} [options]`)}\n`);
+  Logger.log(colors.yellow('Options:'));
+
+  const subcmdOptionsHelp = getSubcommandOptionsHelp(subcmd);
+  Logger.log(subcmdOptionsHelp);
+}
+
+export function verifyOptions(subcommand: string, options: Options): boolean {
+  const optionsPassed = Object.keys(options).filter(option => options[option] !== false);
+  const availableOptions = AVAILABLE_SUBCOMMANDS[subcommand].options;
+
+  const availableOptionsNames = availableOptions.map(option => option.name);
```

Comment:
I don't think that should be an issue here since we haven't decided to use any aliases for the subcommand options. Aliases exists only for the flags (valued options).